.storybook/preview.tsx

```diff
@@ -2,6 +2,7 @@ import * as React from "react";
 import wonderBlocksTheme from "./wonder-blocks-theme";
 import {Decorator} from "@storybook/react";
 import {semanticColor} from "@khanacademy/wonder-blocks-tokens";
+import {initAnnouncer} from "../packages/wonder-blocks-announcer";
```

Comment:
Note: This wasn't working as a package import, I believe because `wonder-blocks-announcer` hasn't yet been published.

Reply:
It should work even without it being published. Is it listed in the appropriate tsconfig and package.json's?

Reply:
I think when I had this issue before, I just had to do a local build for things to start working right.

Reply:
Good tip! I'd looked around but missed a `tsconfig-build.json` file...I _think_ I got it to work with an update. No more red squigglies.

Reply:
Actually this didn't seem to work. The red squigglies are back for me even after a `pnpm install` and `pnpm build` 😭

Reply:
Have you tried running a local build so the built files exist?

Reply:
Is WB on pnpm? It may be that `pnpm i` needs to be run, too.